src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/TimePeriodDataTable.test.tsx

```diff
@@ -264,8 +266,8 @@ describe('TimePeriodDataTable', () => {
       results: [
         {
           filters: ['598ed9fd-b37e-4e08-baec-08d78f6f2c4d'],
-          geographicLevel: 'Country',
-          location: { country: { code: 'E92000001', name: 'England' } },
+          geographicLevel: 'localAuthority',
```

Comment:
Why the change from "country" type to "localAuthority" here?  Is it just to make the grouping functions' test setups easier?

Reply:
Find and replace gone wrong I'm afraid - fixed!